packages/translations/src/i18next/i18next.ts

```diff
@@ -120,7 +121,7 @@ export const initializeTranslations = async () => {
 export const getLanguage = () => i18n.language || initial_language;
 
 // eslint-disable-next-line no-unused-vars
-export const changeLanguage = async (lang, cb) => {
+export const changeLanguage = async (lang: string, cb: (arg0: string) => void) => {
     // TODO: uncomment this when translations are ready
```

Comment:
ashraf can you check what does it mean?
which line do we have to uncommentt? :-D

Reply:
Hey Ako, I think it is time to remove that comment😇 ,It was a comment made 3 years before. Most probably when we started initial translations with multiple translation files.😂

Reply:
would you please remove it? because it affects in the sonarcloud report